managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx

```diff
@@ -0,0 +1,172 @@
+</Concept>
+
+<Concept>
+
+Where can I find my main domain files?
+
+Your main domain files can be located in the `public_html` or `www` directory.
+
+</Concept>
+
+<Concept>
+
+## Where can I find my subdomain files?
+
+Your subdomain files will be located in the `/subdomain.domain.tld` directory (compared to `/subdomain` in the old environment).
+
+</Concept>
+
+<Concept>
+
+## Where can I find my backup files?
+
+Your backup files will not be directly migrated. You will retain access for one month to retrieve them from the old hosting platform.
+
+</Concept>
+
+<Concept>
+
+## What happens to my FTP accounts?
+
+All FTP accounts and configurations will be migrated. The existing password will remain functional, and the hostname will be updated to `ftp.domain.tld` or the platform's hostname in [Scaleway's Web Hosting Overview](https://console.scaleway.com/webhosting/webhosting).
+
+</Concept>
+
+<Concept>
+
+## What happens to my Web Crons?
+
+Your Web Crons will be migrated and can be found in cPanel under **Cron Jobs**.
+
+</Concept>
+
+<Concept>
+
+## What happens to my logs?
+
+Your log files will not be migrated. New log files can be accessed in cPanel at `~/access-logs` or `~/logs`.
+
+</Concept>
+
+<Concept>
+
+## What happens to my web statistics?
+
+Your web statistics will be migrated to cPanel. You will find them under **Awstats**.
+
+</Concept>
+
+<Concept>
+
+What happens to my web applications?
+
+WordPress websites will use WP-Toolkit for application management, while cPanel's Softaculous will be used for installing new web applications.
+
+</Concept>
+
+<Concept>
+
+## What happens to my alias and subdomains?
+
+Your alias domains and subdomains can be managed in cPanel under **Domains**.
+
+</Concept>
+
+<Concept>
+
+## What happens to my emails?
+
+All your emails and mailboxes will be migrated to cPanel.
+
+</Concept>
+
+<Concept>
+
+## What happens to my email redirections and transfers?
+
+All your email redirections and transfers will be migrated to cPanel.
+
+</Concept>
+
+<Concept>
+
+## What happens to my databases?
+
+All your MySQL and PgSQL databases will be migrated to cPanel. The hostname will change and must be updated in the website's configuration files. The hostname can be set as `localhost` if the script is on the same platform or the platform's hostname or IP address.
+
+</Concept>
+
+<Concept>
+
+## What happens to my DNS Configuration (if the nameservers of the domain resolve to ns0.online.net)?
+
+- **Nameservers:** We will attempt to update the nameservers to `ns1.whm.fr-par.scw.cloud` and `ns2.whm.fr-par.scw.cloud`.
+- **DNS Zone:** Migration of DNS Zone records not conflicting with the new cPanel Zone will be attempted (e.g., user-custom DNS records).
+
+</Concept>
+
+<Concept>
+
+When will I be billed for the migrated Web Hosting plan?
+
+Your Scalewy Web Hosting billing will start either at the expiration date of Classic Hosting plan or upon the change of the offer on Scalaeway, whichever comes first.
```

Comment:
This isn't clear to me - what is meant by "upon to change of the offer on Scaleway" - could it be "when you migrate to Scaleway"?